recipes/cpp-subprocess/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2.0.1":
+    url: "https://github.com/nextsilicon/cpp-subprocess/archive/refs/tags/v2.0.1.tar.gz"
```

Comment:
it's a fork of https://github.com/arun11299/cpp-subprocess. There are not so many modifications. Might be better to package 2.0.0, with few patches if it's broken.

Reply:
added version 2.0